content/en/news/releases/1.14.x/announcing-1.14/upgrade-notes/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Istio 1.14 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.14.0.
+publishdate: 2022-05-24
+weight: 20
+---
+
+When you upgrade from Istio 1.13.x to Istio 1.14.0, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.14.0.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio `1.13.x` or Istio `1.12.x`.
```

Comment:
Since we mention both 1.12 and 1.13 in the last sentence of this paragraph, should we have both in the first sentence. If we have both listed, do we also need to include change notes here from going from 1.12 to 1.13? I'm not sure if they are. 
Or maybe it's best to just point to the 1.13 change notes for additional notes if upgrading from 1.12, and then do we remove 1.12 from the last sentence..
```suggestion
When you upgrade from Istio 1.12.x or 1.13.x to Istio 1.14.0, you need to consider the changes on this page.
These notes detail the changes which purposefully break backwards compatibility with Istio 1.14.0.
The notes also mention changes which preserve backwards compatibility while introducing new behavior.
Changes are only included if the new behavior would be unexpected to a user of Istio `1.13.x` or Istio `1.12.x`.
```
or maybe it's
```suggestion
When you upgrade from Istio 1.13.x to Istio 1.14.0, you need to consider the changes on this page.
These notes detail the changes which purposefully break backwards compatibility with Istio 1.14.0.
The notes also mention changes which preserve backwards compatibility while introducing new behavior.
Changes are only included if the new behavior would be unexpected to a user of Istio `1.13.x`.
Users upgrading from 1.12.x to Istio 1.14.0 should also reference the [1.13.0 change logs](/news/releases/1.13.x/announcing-1.13/change-notes/).
```

Reply:
The second one looks better to me.